src/DataLayouts/DataLayouts.jl

```diff
@@ -893,6 +940,12 @@ function IV1JH2{S, Ni}(array::AbstractMatrix{S}) where {S, Ni}
     IV1JH2{S, Ni, typeof(array)}(array)
 end
 
+function replace_basetype(data::IV1JH2{S, Ni}, ::Type{T}) where {S, Ni, T}
```

Comment:
We don't really need this definition as we can only use this datalayout for output / plotting and Duals won't work there